.github/workflows/upload-celocli-executables.yml

```diff
@@ -1,8 +1,19 @@
 name: Upload celocli executables
 
 on:
-  release:
-    types: [released]
+  workflow_call:
+    inputs:
+      ref:
+        description: 'Commit-ish of the developer-tooling repo that the celocli should be compiled and uploaded from'
+        type: string
```

Comment:
AFAIK there's not much more validation we can do that this input is a correct commit.

Reply:
Done in c16d7bfaa